test/EndToEndTests/Tests/Client/AsynchronousTests/AsyncMethodTests.cs

```diff
@@ -110,6 +110,13 @@ public async Task SaveChangesTest()
             var dataServiceResponse = await context.SaveChangesAsync(SaveChangesOptions.BatchWithIndependentOperations | SaveChangesOptions.UseRelativeUri);
             Assert.AreEqual((dataServiceResponse.First() as ChangeOperationResponse).StatusCode, 201, "StatusCode == 201");
 
+            // UseJsonBatch

```

Comment:
Note that this doesn't actually appear to test that the batch is created (or read) as JSON -- just that setting the option doesn't break anything.

Reply:
+1 You actually want to check the both request and response to see if they are json

Reply:
#Resolved